### PR TITLE
Update lambda to openfaas blog post

### PR DESCRIPTION
**Pull request by welteki**
## Description

- Update openfaas functions to use latest node template
- Update cli commands
- Update outdated referencers and links
- Update links to source code

The updated code example can now be found here: https://github.com/welteki/lambda-openfaas-blog

## Motivation and Context

This old post is still popular make it up to date with the latest changes to OpenFaaS.

## Have you applied the editorial and style guide to your post?

See the [README.md](README.md)

## How have you tested the instructions for any tutorial steps?

- Verified the updated OpenFaaS function are building successfully. (did not test updated functions against DynamoDB)
- Verified the updated CLI command.

## Types of changes

- [ ] New blog post
- [x] Updating an existing blog post
- [ ] Updating part of an existing page
- [ ] Adding a new page

## Checklist:

- [x] I have given attribution for any images I have used and have permission to use them under Copyright law
- [x] My code follows the [writing-style of the publication](README.md) and I have checked this
- [x] I've read the [CONTRIBUTION](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md) guide
- [x] I have signed-off my commits with `git commit -s`

### CODE REVIEW

File: _posts/2019-10-08-lambda-to-openfaas.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ Let's see what it takes to upgrade your AWS Lambda functions to OpenFaaS and Ser
 ## Prerequisites
```
Comment (alexellis) [inline _posts/2019-10-08-lambda-to-openfaas.md:26]: I'd suggest putting yourself in as a co author / editor like I've done on some of the collaborative posts we've worked on. That way people know someone is still maintaining this article.

You may also want to add a `> Last Updated: 2023.... ` up top.

File: _posts/2019-10-08-lambda-to-openfaas.md [hunk 2]
```diff
@@ -47,27 +47,28 @@ exports.handler = async (event) => {
 };
 ```
 
-Let's examine the example for OpenFaaS. When you create a new function with the OpenFaaS CLI and the `node10-express` template, sample code is provided as shown here:
+Let's examine the example for OpenFaaS. When you create a new function with the OpenFaaS CLI and the `node17` template, sample code is provided as shown here:
```
Comment (alexellis) [inline _posts/2019-10-08-lambda-to-openfaas.md:50]: Did you decide against LTS and node18?

Comment (welteki) [reply _posts/2019-10-08-lambda-to-openfaas.md:63]: Will update this to node18

**Comment by alexellis** [comment]
This is still showing node17 - can you switch it to the new node18 template now please?

**Comment by welteki** [comment]
Switched to node18

**Review by alexellis — APPROVED** [review]
Approved

**Pull request merged**